selfdrive/locationd/locationd.cc

```diff
@@ -131,7 +131,7 @@ void Localizer::build_live_location(cereal::LiveLocationKalman::Builder& fix) {
 
   // write measurements to msg
   init_measurement(fix.initPositionGeodetic(), fix_pos_geo_vec, nans, true);
```

Comment:
Also set flag on positionGeodetic since it's also wrong

Reply:
Done